src/db/options.rs

```diff
@@ -55,8 +55,7 @@ pub struct CreateCollectionOptions {
     /// Specifies a validator to restrict the schema of documents which can exist in the
     /// collection. Expressions can be specified using any query operators except `$near`,
     /// `$nearSphere`, `$text`, and `$where`.
-    #[serde(rename = "validator")]
-    pub validation: Option<Document>,
+    pub validator: Option<Document>,
```

Comment:
This was originally a typo I believe, all other drivers have this as `validator`. Took the opportunity to fix it up while I'm modifying these files.

Reply:
not sure what you've got in mind in terms of a change log/migration guide for 2.0, but would it make sense to file a separate ticket for this just so we remember to mention it?